config/engines.yml

```diff
@@ -249,6 +249,16 @@ rubymotion:
   enable_regexps:
   default_ratings_paths:
     - "**.rb"
+
+shellcheck:
```

Comment:
I matched builder's assuming that one was sorted; guess not. Will fix here at least.